common/djangoapps/django_sudo_helpers/decorators.py

```diff
@@ -31,24 +32,25 @@ def sudo_required(func_or_region):
     def wrapper(func):  # pylint: disable=missing-docstring
         @wraps(func)
         def inner(request, *args, **kwargs):    # pylint: disable=missing-docstring
-            course_specific_region = kwargs.get('course_id')
-            if 'course_key_string' in kwargs:
-                course_specific_region = kwargs.get('course_key_string')
-            if 'library_key_string' in kwargs:
-                course_specific_region = kwargs.get('library_key_string')
-
-            # N.B. region is captured from the enclosing sudo_required function
-            if not request.is_sudo(region=region or course_specific_region):
-                response_format = request.REQUEST.get('format', 'html')
-                if (response_format == 'json' or
-                        'application/json' in request.META.get('HTTP_ACCEPT', 'application/json')):
-                    return JsonResponse({'error': 'Unauthorized'}, status=401)
-
-                return redirect_to_sudo(request.get_full_path(), region=region or course_specific_region)
-
-            if RESET_TOKEN is True:
-                # Provide new sudo token content and reset timeout on activity
-                new_sudo_token_on_activity(request, region=region or course_specific_region)
+            if settings.FEATURES.get('ENABLE_DJANGO_SUDO', False):
```

Comment:
if you want to avoid all the indentation below, you can instead do something like 
```python
if not settings.FEATURES.get('ENABLE_DJANGO_SUDO', False):
    # noop
    return
```